libs/dh/shared/ui-paginator/README.md

```diff
@@ -0,0 +1,40 @@
+# Paginator Ui
+
+Used for pagination of tables. The component includes translations and consistent behaviour of pagination.
+
+## Usage
+
+1. Import the `DhSharedUiPaginatorComponent` inside a SCAM module or a standalone component:
+
+   ```ts
+   imports: [DhSharedUiPaginatorComponent];
+   ```
+
+2. Add `<dh-shared-ui-paginator></dh-shared-ui-paginator>` to the template.
+
+3. Get the reference to the paginator instance:
+
+   ```ts
+   @ViewChild(DhSharedUiPaginatorComponent) paginator!: DhSharedUiPaginatorComponent;
+   ```
+
+4. Connect the data source with the paginator:
+
+   ```ts
+   ngAfterViewInit() {
+     this.dataSource.paginator = this.paginator.getPaginator();
```

Comment:
If you decide to change the function call as mentioned below, then don't forget to update the docs here 😄